FRAuth/FRAuth/DeviceBinding/ApplicationPinDeviceAuthenticator.swift

```diff
@@ -22,15 +23,34 @@ open class ApplicationPinDeviceAuthenticator: DeviceAuthenticator, CryptoAware {
     var cryptoKey: CryptoKey?
     /// AppPinAuthenticator to take care of key generation
     var appPinAuthenticator: AppPinAuthenticator?
+    /// ApplicationPinDeviceAuthenticatorDelegate for collection the Pin
+    public var delegate: ApplicationPinDeviceAuthenticatorDelegate?
```

Comment:
To avoid confusion I would suggest that we rename this to something like 

`public var collector: ApplicationPinDeviceAuthenticatorDelegate?`